.pre-commit-config.yaml

```
@@ -0,0 +1,40 @@
+        exclude: ".*.md"
+      - id: check-yaml
+      - id: check-toml
+      - id: check-added-large-files
+  - repo: https://github.com/rhysd/actionlint
+    rev: v1.7.4
+    hooks:
+      - id: actionlint
+  - repo: https://github.com/codespell-project/codespell
+    rev: v2.2.4
+    hooks:
+    - id: codespell
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)